components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -207,6 +213,17 @@ function matchFunctions.getExtraData(match)
 	return match
 end
 
+function getPlayerFlag(name)
+	local data = mw.ext.LiquipediaDB.lpdb('player', {
+		conditions = '[[pagename::' .. mw.ext.TeamLiquidIntegration.resolve_redirect( name ):gsub('%s+', '_') .. ']]',
+		query = 'nationality'
+	})
+	if type(data) == 'table' and data[1] then
+		return data[1]['nationality']:lower()
+	end
+	return 'world'
```

Comment:
Spontaneously, personally I prefer no flag over than a world flag.

Reply:
I wanted to encourage editors to fill in the flag by showing one nevertheless

Reply:
Actually, come to think about it. Isn't it better to query the Broadcaster table on the tournament instead?

Reply:
I didn't know that was possible

Reply:
https://liquipedia.net/rocketleague/Special:LiquipediaDB/Rocket_League_Championship_Series/2021-22/Winter#broadcasters